### ISSUE TITLE
timeouts after upgrade to latest

### ISSUE THREAD

**Issue by xsnrg** [aaronwmorris/indi-allsky#1049]
My camera recently stopped working, so as part of getting it going again, I figured I would upgrade the software to play with some cool new features. 

After running the update, I needed to manually update `libxisf` for some elements missing in indi_asi_ccd, and then everything indi started. 

My remaining error that seems harder to track down is 

```
Dec  4 12:00:55 rpi4 [ERROR] MainProcess allsky.unhandled_exception() #35: Type: <class 'indi_allsky.exceptions.TimeOutException'>
Dec  4 12:00:55 rpi4 [ERROR] MainProcess allsky.unhandled_exception() #36: Value: Timeout finding control TELESCOPE_INFO
Dec  4 12:00:55 rpi4 [ERROR] MainProcess allsky.unhandled_exception() #41: '  File "indi-allsky/allsky.py", line 119, in <module>\n    action_func(*args_list, **kwargs_dict)\n'
Dec  4 12:00:55 rpi4 [ERROR] MainProcess allsky.unhandled_exception() #41: '  File "indi-allsky/indi_allsky/allsky.py", line 1026, in run\n    self._initialize()\n'
Dec  4 12:00:55 rpi4 [ERROR] MainProcess allsky.unhandled_exception() #41: '  File "indi-allsky/indi_allsky/allsky.py", line 577, in _initialize\n    self.indiclient.configureTelescopeDevice(telescope_config)\n'
Dec  4 12:00:55 rpi4 [ERROR] MainProcess allsky.unhandled_exception() #41: '  File "indi-allsky/indi_allsky/camera/indi.py", line 724, in configureTelescopeDevice\n    self.configureDevice(self._telescope_device, *args, **kwargs)\n'
Dec  4 12:00:55 rpi4 [ERROR] MainProcess allsky.unhandled_exception() #41: '  File "indi-allsky/indi_allsky/camera/indi.py", line 703, in configureDevice\n    self.set_number(device, k, v)\n'
Dec  4 12:00:55 rpi4 [ERROR] MainProcess allsky.unhandled_exception() #41: '  File "indi-allsky/indi_allsky/camera/indi.py", line 1230, in set_number\n    c = self.get_control(device, name, \'number\')\n'
Dec  4 12:00:55 rpi4 [ERROR] MainProcess allsky.unhandled_exception() #41: '  File "indi-allsky/indi_allsky/camera/indi.py", line 1221, in get_control\n    raise TimeOutException(\'Timeout finding control {0}\'.format(name))\n'
```
Any help appreciated.

More info:

```$ apt list --installed|grep indi-asi

indi-asi/jammy,now 2.3+t202312031408~ubuntu22.04.1 arm64 [installed]
```
```
$ apt list --installed|grep libxisf

libxisf/jammy,now 0.2.10~202312021728~ubuntu22.04.1 arm64 [installed,automatic]
```
TELESCOPE_TRACK_STATE does get set to `off` earlier in the log, so the INDI commands to telescope do seem to otherwise be working.

**Comment by aaronwmorris** [comment]
Looks like some telescope attributes were removed in indi 2.0.5.

Merged #1051 to fix.

**Comment by xsnrg** [comment]
pulling... updating... fixed. Thanks!
